opcpa_tpr_config/xpm_prog.py

```diff
@@ -1,18 +1,47 @@
 import argparse
+import copy
 import itertools
 
 import numpy as np
 from psdaq.cas.pvedit import Pv
 from psdaq.seq.seq import Branch, ControlRequest, FixedRateSync
 from psdaq.seq.seqprogram import SeqUser
 
-factors = [2, 2, 2, 2, 5, 5, 5, 5, 7, 13]  # 910,000
+all_factors = [1, 2, 2, 2, 2, 5, 5, 5, 5, 7, 13]  # 910,000
 carbide_factors = [1, 2, 2, 5, 5, 5, 5, 13]  # 32,500 (remove 2, 2, 7, add 1)
+carbide_70k_factors = [1, 2, 2, 5, 5, 5, 5]  # 70,000 (remove 13, add 1)
```

Comment:
Being rather uninformed w.r.t this laser config factors, it looks like you just removed a 13, and didn't add a 1?  Same with the above `carbide_factors`